Tasks/AzureWebAppV1/deploymentProvider/AzurePublishProfileDeploymentProvider.ts

```diff
@@ -0,0 +1,110 @@
+import tl = require('azure-pipelines-task-lib/task');
+import { KuduServiceUtility } from 'azurermdeploycommon/operations/KuduServiceUtility';
+import { Kudu } from 'azurermdeploycommon/azure-arm-rest/azure-arm-app-service-kudu';
+import { PackageType } from 'azurermdeploycommon/webdeployment-common/packageUtility';
+import { TaskParameters } from '../taskparameters';
+import { IWebAppDeploymentProvider } from './IWebAppDeploymentProvider';
+var zipUtility = require('azurermdeploycommon/webdeployment-common/ziputility.js');
+var deployUtility = require('azurermdeploycommon/webdeployment-common/utility.js');
+
+var parseString = require('xml2js').parseString;
+
+interface scmCredentials {
```

Comment:
this is appearing multiple times. should definitely move to common model file

Reply:
the common libraries are different for both Azure WebDeploy task and App service manage task. Thus can't be put into common library.